tests/conftest.py

```diff
@@ -7,14 +7,18 @@
 from sql.magic import SqlMagic, _set_sql_magic
 from sql import connection
 
-# currently the connection widget loads the config when importing the module
-# so we need to ensure the magic is initialized before importing the widget
-shell = InteractiveShell()
-sql_magic = SqlMagic(shell)
 
-# set the default dsn filename to odbc.ini so we don't read from the home directory
-sql_magic.dsn_filename = "jupysql-plugin.ini"
-_set_sql_magic(sql_magic)
+def _init_sql_magic():
+    # currently the connection widget loads the config when importing the module
+    # so we need to ensure the magic is initialized before importing the widget
+    shell = InteractiveShell()
+    sql_magic = SqlMagic(shell)
+
+    # set the default dsn filename to odbc.ini so we don't read from the home directory
```

Comment:
I think it should be `set the default dsn filename to jupysql-plugin.ini` ?

Reply:
fixed https://github.com/ploomber/jupysql-plugin/pull/67/commits/7da3f22c1f7270de6589837db7f8496ef50f00b9